Library/Homebrew/os/mac/mach.rb

```diff
@@ -51,8 +51,10 @@ def mach_data
     end
   end
 
-  def dynamically_linked_libraries
-    macho.linked_dylibs
+  def dynamically_linked_libraries(except: [])
+    lcs = macho.dylib_load_commands.reject { |lc| [*except].include?(lc.type) }
```

Comment:
I did this to allow someone to pass either an array of symbols to filter on or a single symbol. The `[*except]` will correctly pack either a single value or an array.

That's pretty hard to read though, so I could replace it with something like this:

```ruby
unless except.is_a? Array
  except = [except]
end
```